wasm/jit/jit_amd64.go

```diff
@@ -1096,6 +1098,90 @@ func (b *amd64Builder) handleLe(o *wazeroir.OperationLe) error {
 	return nil
 }
 
+func (b *amd64Builder) handleLoad(o *wazeroir.OperationLoad) error {
+	base := b.locationStack.pop()
+	if base.onStack() {
+		if err := b.moveStackToRegisterWithAllocation(generalPurposeRegisterTypeInt, base); err != nil {
```

Comment:
how about this?

```suggestion
		if err := b.moveStackToRegisterWithAllocation(generalPurposeIntRegister, base); err != nil {
```

Reply:
I prefer as-is because the prefix `generalPurposeRegisterType*` is actually the type name :D